Evaluation.css

```diff
@@ -0,0 +1,105 @@
+body{
+	font-family: "Arial", sans-serif;
+	position: relative;
+  margin: 0;
+  padding-bottom: 6rem;
+  min-height: 100%;
+
+}
+#pageHeader{
+	font-weight: 400;
+	font-size: 24px;
+	list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    width: 100%;
+    box-shadow: 0px 2px 2px #888888;
+}
+
+#pageHeader li{
+	display: inline-block;
+	padding-left: 2vw;
+	padding-right:2vw;
+}
+
+
+#pageHeader>li>a{
+	display: block;
+	color: #414141;
+	text-align: center;
+	padding: 1vw;
+	text-decoration: none;
+}
+.right-button{
+	float:right;
+}
+#main-logo{
+	height: 35px; 
+	vertical-align: middle;
+}
+
+#active-page{
+	border-style: solid;
+	border-width: 0px 0px 5px 0px;
+	border-color: #03a8f3;
+}
+
+/*table, td, th {
+    border: 1px solid black;
+}
+*/
+.tableBlock{
+	border-collapse: collapse;
+	border: 0px solid;
+	
+	width: 1200px;
+	top:0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	margin: auto;
+	color: #414141;
+
+}
+.tableBlock  th{
+	font-size: 22px;
+	background-color: #03a8f3;
+	color: #ffffff;
+	height: 4em;
+	font-weight: 500;
+	text-align: left;
+}
+.hidden-clone-row{
+	width:50px;
+
+}
+.hidden-clone-row input{
+	 outline: none;
+}
+.tableBlock td{
+	font-size: 18px;
+	vertical-align: center;
+}
+.Detail{
+	text-align: right !important;
+}
+td{
+  border-bottom:1pt solid black;
+}
+
+.BottomFooter{
+	bottom:0;
+    position:fixed;
+    z-index:150;
+    _position:absolute;
+    _top:expression(eval(document.documentElement.scrollTop+
```

Comment:
I'm curious as to what you're trying to achieve here. Are there simpler solutions available?